### PR TITLE
terraform: Use for_each on github resources

### PR DESCRIPTION
**Pull request by phy1729**
Use the `for_each` meta-argument rather than many separate resources. While this will have a heck of a delta `terraform plan`-wise initially, after doing the necessary `terraform state mv` commands there should be no delta.

### CODE REVIEW

**Comment by the-maldridge** [comment]
There is one new untracked repo that was added just slightly before your PR.  Please add an additional repo for xmandump and writeable to pkg-committers.

**Comment by phy1729** [comment]
Done.

File: terraform/github.tf [hunk 1]
```diff
@@ -4,299 +4,191 @@ provider "github" {
...
+    }
+
+    musl-fts = {
+      description = "Implementation of fts(3) for musl libc packages in Void Linux"
+      teams = [
+        "pkg-committers",
+      ]
+    }
+
+    openbsd-man = {
```
Comment (the-maldridge) [inline terraform/github.tf:159]: This is an archived repository, ideally it would stay that way.

File: terraform/github.tf [hunk 2]
```diff
@@ -4,299 +4,191 @@ provider "github" {
...
+      description = "The OpenBSD man(1) utility for Linux"
+      teams = [
+        "pkg-committers",
+      ]
+    }
+
+    xmandump = {
```
Comment (the-maldridge) [inline terraform/github.tf:166]:
```
~ resource "github_repository" "repositories" {
      ~ allow_merge_commit = false -> true
        allow_rebase_merge = true
      ~ allow_squash_merge = false -> true
```

**Comment by phy1729** [comment]
Done and done. Rebased to keep the xmandump changes together.

**Comment by the-maldridge** [comment]
Merged.  Thanks for your contribution!

**Pull request merged**